src/pages/index.md

```diff
@@ -1,70 +1,26 @@
 ---
 templateKey: index-page
-title: Great coffee with a conscience
+title: Building diversity in STEM
 image: /img/home-jumbotron.jpg
-heading: Great coffee with a conscience
-subheading: Support sustainable farming while enjoying a cup
+heading: Building diversity in STEM
+subheading: Education and opportunities for underserved youth
 mainpitch:
-  title: Why Kaldi
+  title: Why Pi515
   description: >
-    Kaldi is the coffee store for everyone who believes that great coffee
-    shouldn't just taste good, it should do good too. We source all of our beans
-    directly from small scale sustainable farmers and make sure part of the
-    profits are reinvested in their communities.
+    We go further than just the technical skills that our students deserve. Our students receive supplementary skills in College Prep, Financial Literacy, and Entrepreneurship. These vital skills are taught by our program to amplify our impact to maximize success.
```

Comment:
>College Prep, Financial Literacy, and Entrepreneurship
Are these supposed to be capitalized?

Reply:
I was going to give them emphasis, but nah